test/Test/P/List.hs

```diff
@@ -19,6 +19,10 @@ prop_lastMaybe_empty :: Property
 prop_lastMaybe_empty =
   lastMaybe [] === (Nothing :: Maybe ())
 
+prop_count :: [Int] -> Property
+prop_count list =
+  count (>= 0) list === length (filter (>= 0) list) 
+
```

Comment:
I think more generic properties for this function should include checks on the output for `(===) <$> count (const True) <*> id` and `count (const False)`.